assets/less/zenodo-rdm/collections/form.overrides

```diff
@@ -0,0 +1,12 @@
+
+form[name="register_user_form"],
```

Comment:
is there any other way we could match these elements, fe by class name ? attribute selectors tend to be less efficient